### PR TITLE
[Feature] Semi automated with label studio

### PR DESCRIPTION
**Pull request by fengshiwest**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

[Feature] support semi-automatic annotation base Label-Studio

## Modification

- projects/LabelStudio/backend_template/mmdetection.py
- projects/LabelStudio/backend_template/_wsgi.py
- corresponding user guide documents

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by zytx121** [comment]
Hi @fengshiwest, excellent work! Thank you for your PR, which further expands the application scenarios of MMRotate

**Review by zytx121 — DISMISSED** [review]
LGTM.

File: projects/LabelStudio/backend_template/mmdetection.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+from label_studio_tools.core.utils.io import get_data_dir
+from mmdet.apis import inference_detector, init_detector
+
+from mmrotate.structures.bbox import RotatedBoxes
+
+logger = logging.getLogger(__name__)
+
+
+class MMDetection(LabelStudioMLBase):
+    """Object detector based on https://github.com/open-mmlab/mmdetection."""
```
Comment (liuyanyi) [inline projects/LabelStudio/backend_template/mmdetection.py:23]: Change the name and url to MMRotate

Comment (fengshiwest) [reply projects/LabelStudio/backend_template/mmdetection.py]: Have updated!

**Comment by RangiLyu** [comment]
Please resolve the conflict.

**Review by RangiLyu — APPROVED** [review]

**Pull request merged**